src/App.tsx

```diff
@@ -28,45 +28,62 @@ export const App = () => {
   const [selectedUser, setSelectedUser] = useState('0');
   const [submittedUser, setSubmittedUser] = useState(true);
 
-  const submitHandle = (event: React.FormEvent) => {
+  const handleSubmit = (event: React.FormEvent) => {
     event.preventDefault();
-    if (inputText.trim() !== '') {
-      setSubmittedTitle(true);
-    } else {
+    if (inputText.trim() === '' && selectedUser === '0') {
```

Comment:
Optimize code:
```js
if (condition1 || condition2) {
  setState1(condition1);
  setState2(condition2);

  return;
}
```
You can skip a lot of duplicate code.

Reply:
Sorry, I can't understand your tip. In this task, I must throw an error (near the field) when the field is empty or Select is selected. So without these statements, it will not work. Because I need to set the Error in different situations:

- when the first field is empty;
- when the second field is empty;
- when both fields are empty.

Reply:
Your error depends on condition. If one if your conditions is true you can set errors and exit the function. But you did it in 3 ifs. Combine it in one.
Example:
```js
const a = '';
const b = 1;

if (a === '' || b === 0) { // you have any combination of errors
  setErrorA(a === ''); // true
  set ErrorB(b === 0) // false

  return;
}
```

Reply:
OMG, I got it, thank you very much for the explanation, it was so easy...